nengo_viz/static/viz_value.js

```diff
@@ -17,7 +17,8 @@ VIZ.Value = function(parent, sim, args) {
     this.display_time = args.display_time;
 
     /** for storing the accumulated data */
-    this.data_store = new VIZ.DataStore(this.n_lines, this.sim, 0.01);
+    var synapse = (args.synapse !== null) ? args.synapse : 0.01;
```

Comment:
synapse can't be null here, since it's always being set to either 0 or 0.01 above.  And if it's not set, it'll be undefined, not null.